src/pages/index.js

```diff
@@ -11,6 +11,9 @@ export default function Home() {
   const [play] = useSound(laugh1)
   const [reverse, setReverse] = useState(false)
 
+
+  var ourTopChildrenLaughs = [play1, play2, play3]
+  // when you say "initiate a const", do you mean write a const? Or something more fancy?
```

Comment:
I mean `const ourTopChildrenLaughs`.

But before you can do `[play1, play2, play3]` you need to make them by `const [play1] = useSound(laugh1]`, `const [play2] = useSound(laugh2)` etc.